android/app/src/main/java/app/organicmaps/widget/placepage/PlacePageUtils.java

```diff
@@ -79,7 +80,11 @@ static String toString(@BottomSheetBehavior.State int state)
   public static void copyToClipboard(Context context, View frame, String text)
   {
     Utils.copyTextToClipboard(context, text);
-    if (Build.VERSION.SDK_INT < Build.VERSION_CODES.TIRAMISU)
+
+    KeyguardManager keyguardManager = (KeyguardManager) context.getSystemService(Context.KEYGUARD_SERVICE);
+
+    //only show snackBar if device is locked or android sdk is earlier than TIRAMISU.
+    if (Build.VERSION.SDK_INT < Build.VERSION_CODES.TIRAMISU || Build.VERSION.SDK_INT > Build.VERSION_CODES.TIRAMISU && keyguardManager.isDeviceLocked())
```

Comment:
Please, put braces, the conditions order is not obvious here:
```if (Build.VERSION.SDK_INT < Build.VERSION_CODES.TIRAMISU || Build.VERSION.SDK_INT > Build.VERSION_CODES.TIRAMISU && keyguardManager.isDeviceLocked())```

- What's wrong with ```Build.VERSION.SDK_INT == Build.VERSION_CODES.TIRAMISU```? It doesn't support a snack bar at all?
- ```Build.VERSION.SDK_INT > Build.VERSION_CODES.TIRAMISU``` supports a snack bar in locked mode only?

Reply:
apologies, working on the fix.

Reply:
how about this modification: 
``` 
if(Build.VERSION.SDK_INT < Build.VERSION_CODES.TIRAMISU)
      {
        Utils.showSnackbarAbove(view.getRootView().findViewById(R.id.pp_buttons_layout), view,
                                ctx.getString(R.string.copied_to_clipboard, phoneNumber));
      }
      else if (keyguardManager.isDeviceLocked())
      {
        Utils.showSnackbarAbove(view.getRootView().findViewById(R.id.pp_buttons_layout), view,
                                ctx.getString(R.string.copied_to_clipboard, phoneNumber));
      }
 ```

Reply:
```
if(Build.VERSION.SDK_INT < Build.VERSION_CODES.TIRAMISU || keyguardManager.isDeviceLocked())
      {
        Utils.showSnackbarAbove(view.getRootView().findViewById(R.id.pp_buttons_layout), view,
                                ctx.getString(R.string.copied_to_clipboard, phoneNumber));
      }
```

Reply:
this is so much concise, you should create this pull request.